src/components/FavoritePage.vue

```diff
@@ -39,14 +41,24 @@
 
 <script>
 import FinishModal from './FavoriteFinishModal.vue'
-import errorImage from '../assets/css/errorImage.png'
+
+const ImageIndex = {
+  left: 0,
+  right: 1
+}
 
 export default {
   components: {
     FinishModal
   },
+  data() {
+    return {
+      imagePair: [false, false],
+      imageIndex: ImageIndex
```

Comment:
use `Computed` when using enum type

Reply:
Does it resolve?

Reply:
yes

Reply:
but It's still in the data, and it seems didn't use computed

Reply:
I will merge first, and fix it later